090-Examples/015-prune-your-follows.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Prune your follows powered by Xata
+navTitle: Prune your follows
+keywords: ['examples']
+description: Create an app that unfollows inactive accounts
+slug: examples/pruneyourfollows
+published: false
```

Comment:
The docs page for this app doesn't show in the examples sidepane (where the Gallery is)

https://xata-docs-pr-155.vercel.app/docs/examples/pruneyourfollows

Should we flip the published flag to true here?